ValidateBST.java

```diff
@@ -0,0 +1,68 @@
+// Time Complexity : O(h) height of tree, if skewed then worst case iterate all
+// Space Complexity : Not using additional list to store, just constant variables
```

Comment:
Please check the Time and Space complexity.

Reply:
@shrivatsav Updated the complexity for ValidateBST.java.
// Time Complexity : O(n) ~ O(h) h - height of tree because if skewed then worst case visit all nodes
// Space Complexity : O(n) function stack size; depth of tree nodes. If no function stack size considered then O(1) for constant variables